pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -669,19 +669,7 @@ func (h *Helper) ServerPrereleaseVersion() (semver.Version, error) {
 		return semver.Version{}, fmt.Errorf("did not find version in command output")
 	}
 
-	// The OSE version may have > 4 parts to the version string
-	// We'll only take the first 3
-	parts := strings.Split(versionStr, ".")
-	if len(parts) > 3 {
-		versionStr = strings.Join(parts[:3], ".")
-	}
```

Comment:
@bparees so the issue is that for ocp releases, we don't have a semver conformant version string (it's got 4 numbers instead of 3, like 3.5.0.5). For most comparisons within cluster up, having only the first 3 numbers is ok, even with pre-release versions. We also remove the pre-release part anyway.